HealthcareSystem/GraphicalEditorServerTests/DataFactory/CreateEquipmentType.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Backend.Model.Manager;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GraphicalEditorServerTests.DataFactory
+{
+    public class CreateEquipmentType : ICreateTestObjectFactory<EquipmentType>
+    {
+        public EquipmentType CreateInvalidTestObject()
+        {
+            return new EquipmentType("fsdafafas",false);
```

Comment:
Why is this not valid?

Reply:
I thought that name is not valid, or doesnt mean anything, but l changed that too

Reply:
That does make sense semantically, it's just something we couldn't possibly actually validate.